.github/workflows/update-image.yaml

```
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: 1.20
```

```suggestion
          go-version: "1.20"
```

It is recognized as '1.2’